### PR TITLE
Update uzbek.json

### PR DESCRIPTION
**Pull request by AlJavhar**
I corrected some wrong words and removed unnecessary ones. I will be glad if I see my language corrected on such sites. Thanks a lot!

### Description

Closes #

### CODE REVIEW

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4736191912

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4736283083

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4736337594

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4736629105

**Comment by rizwanmustafa** [comment]
@AlJavhar It seems to me that this PR contains the changes you made in #4194. If my assumption is correct, please close the #4194 PR and continue your work on this PR.

**Comment by rizwanmustafa** [comment]
As far as this PR is concerned, your changes have introduced some duplicate words:

`eshik`, `do’kon`, `bit`, `pul`, `tuxum`, `gul`, `harakat`, `quloq`.

Please remove them, so that the PR checks can be passed.

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]
As far as this PR is concerned, your changes have introduced some duplicate words:

`eshik`, `do’kon`, `bit`, `pul`, `tuxum`, `gul`, `harakat`, `quloq`.

Please remove them, so that the PR checks can be passed.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4742087629

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4742199991

File: frontend/static/languages/uzbek.json [hunk 1]
```diff
@@ -59,15 +59,15 @@
     "taqsimlash",
     "yaratish",
     "meva",
-    "ildiz",
-    "hodisa",
+    "osmon",
+    "Javhar",
```
Comment (Miodec) [inline frontend/static/languages/uzbek.json:63]: Did you just add your name to the word list?

Comment (rizwanmustafa) [reply frontend/static/languages/uzbek.json:63]: image

Comment (Miodec) [reply frontend/static/languages/uzbek.json:63]: image

Why is it capitalised then

Comment (rizwanmustafa) [reply frontend/static/languages/uzbek.json:63]: Idk why. Maybe he capitalized it accidentally because it's his name

Comment (AlJavhar) [reply frontend/static/languages/uzbek.json:63]: This is the Uzbek word. And I would like to be displayed in such a global project))

**Comment by AlJavhar** [comment]
Fixed Uzbek language

**Review by AlJavhar — COMMENTED** [review]
I removed all duplicates and similar

File: frontend/static/languages/uzbek.json (hunk 1 above)
Comment (AlJavhar) [reply frontend/static/languages/uzbek.json:63]: no, pseudonym!

Comment (Miodec) [reply frontend/static/languages/uzbek.json:63]: Still, *your* pseudonym

**Pull request merged**